features/groups/screens/group-members-list.screen.tsx

```diff
@@ -55,15 +58,30 @@ const List = memo(function List() {
   const insets = useSafeAreaInsets()
   const currentSender = useSafeCurrentSender()
   const { xmtpConversationId } = useRouteParams<"GroupMembersList">()
-
-  const { members, isLoading: isLoadingMembers } = useGroupMembers({
+  
+  const { members, sortedMemberIds, isLoading } = useGroupMembersWithSorting({
     xmtpConversationId,
     clientInboxId: currentSender.inboxId,
     caller: "GroupMembersListScreen",
   })
 
-  if (isLoadingMembers) {
-    return null
+  // If members list is empty, trigger a reload
+  useEffect(() => {
+    if (!isLoading && (!members?.ids?.length || sortedMemberIds.length === 0)) {
+      refetchGroupQuery({
+        clientInboxId: currentSender.inboxId,
+        xmtpConversationId,
+        caller: "GroupMembersListScreen-EmptyListReload",
+      }).catch(captureError)
+    }
+  }, [isLoading, members?.ids?.length, sortedMemberIds.length, currentSender.inboxId, xmtpConversationId])
```

Comment:
hmm not sure about that. that's why we should not handle the error in the queryFn have just have a good retry mechanism instead with react-query. does it really happens that the list is empty? I would just ignore for now TBH and maybe we just put `retry: true` in `react-query.client.ts`